GliaWidgets/SecureConversations/SecureConversations.Availability.swift

```diff
@@ -49,16 +49,20 @@ extension SecureConversations {
                 environment.log.warning("Queue ID array for Secure Messaging contains invalid queue IDs: \(invalidIds).")
             }
 
-            guard !queueIds.isEmpty else {
-                // If no queueId is passed, then check default queues instead
+            let matchedQueues = queues.filter { queueIds.contains($0.id) }
+
+            guard !matchedQueues.isEmpty else {
+                environment.log.warning("Provided queue IDs do not match with any queue.")
```

Comment:
It will warn even if an integrator passes the empty array of queues.

Reply:
Fixed f1aecca779c3a677bcbbbbebc079a940130327d7